modnet/hyper_opt/fit_genetic.py

```diff
@@ -33,25 +34,35 @@ def __init__(
             weights (Dict[str, float]): Optional (for joint learning only). The relative loss weights to apply for each target.
         """
 
+        self.act = "elu"
+        self.loss = "mae"
+        self.n_neurons_first_layer = 32 * random.randint(1, 10)
```

Comment:
I wonder if we should use a fixed/customisable seed here so that fit genetic can be made somewhat deterministic?

Reply:
Would this make it deterministic though? Since the models will in any case lead to somewhat slightly different results each time they are fitted, they might differ from iteration to iteration?